modules/arrow/src/arrow-worker-loader.js

```diff
@@ -1,10 +1,11 @@
-// The bundled worker is imported as an inline string
-import worker from '../dist/arrow-loader.worker.js';
-
+/* global __VERSION__ */
 export default {
   name: 'Apache Arrow',
   extensions: ['arrow'],
   mimeType: 'application/octet-stream',
   category: 'table',
-  worker
+  worker: true,
+  defaultOptions: {
+    workerUrl: `https://unpkg.com/@loaders.gl/arrow@${__VERSION__}/dist/arrow-loader.worker.js`
```

Comment:
Idea: Looks like these strings could be auto-deduced inside core based on a `loader.id` or `loader.package` field?

That way we could contain the proliferation of `__VERSION__` and just handle it in core.

Reply:
This way it is possible for users to pass in a custom url pointing to their own server. I actually think this logic is easier to read than putting some complex resolution logic into the core.

Reply:
:+1: completely agree that code should be optimized for clarity, and I do like the fact that this makes things so explicit. 

However, still worth a discussion:
- I do find injected constants confusing (it can take reader quite a while to understand where `__VERSION__` is coming from. A one-line comment above each injection could solve that.
- I usually end up having to deal with side effects from such injections in certain configurations. You already handle the test case via explicit overrides, but how about the `start-local` case? 
- What will `__VERSION__` be in that case, and is there a way for us to still override it in core without having more structural understanding of how these urls are formed?

Reply:
Added comment and fixed `start-local` (inject `__VERSION__: 'latest'`).

We could come up with smarter ways to deal with the local test scenario in the test harness. I prefer not to do it inside the core to avoid unintended implicit behavior.